src/utils/isEmptyList.test.ts

```diff
@@ -0,0 +1,41 @@
+import { isEmptyList } from '../utils';
+
+test('isEmptyList on empty string', () => {
+    expect(isEmptyList('')).toBe(false);
```

Comment:
The expected behaviour here is that `isEmptyList` must return true for all non-array type values.
The aim is to check for empty list, and not to check whether input value is a valid array or not.